Common/UnitDefinitions/HeatTransferCoefficient.json

```diff
@@ -33,14 +33,26 @@
       ]
     },
     {
-      "SingularName": "BtuPerSquareFootDegreeFahrenheit",
-      "PluralName": "BtusPerSquareFootDegreeFahrenheit",
+      "SingularName": "BtuPerHourSquareFootDegreeFahrenheit",
+      "PluralName": "BtusPerHourSquareFootDegreeFahrenheit",
       "FromUnitToBaseFunc": "{x} * 5.6782633411134878",
       "FromBaseToUnitFunc": "{x} / 5.6782633411134878",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "Btu/ft²·hr·°F" ]
+          "Abbreviations": [ "Btu/h·ft²·°F", "Btu/ft²·h·°F", "Btu/hr·ft²·°F", "Btu/ft²·hr·°F" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilocaloriePerHourSquareMeterDegreeCelsius",
+      "PluralName": "KilocaloriesPerHourSquareMeterDegreeCelsius",
```

Comment:
Add CaloriesPerHour... instead, then add "Kilo" as prefix to it. It makes it easier to add other prefixes later.